### PR TITLE
update the readme:

### PR DESCRIPTION
**Pull request by jakevdp**
Please add my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ Hey, we can even edit from github itself!!
 Hola a todos
 
 Ohayoo gozaimasu
+
+Pardon my french
```
Comment (amandalynne) [inline README.md]: I hate this!

Comment (amandalynne) [reply README.md]: Delete it immediately!

**Comment by amandalynne** [comment]
No Way! I don't like it

**Comment by jakevdp** [comment]
why not?

**Comment by amandalynne** [comment]
I change my mind. I like it.

**Pull request merged**